.gitignore

```diff
@@ -31,3 +31,6 @@ node_modules
 
 # Optional REPL history
 .node_repl_history
+
+# Webstorm
```

Comment:
my $.02 is developer specific files should go into  ~/.gitignore_global not the repo's gitignore...not asking for a change, just pointing it out.